README.md

```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://travis-ci.org/jembi/openhim-console.svg?branch=master)](https://travis-ci.org/jembi/openhim-console) [![OpenHIM Core](https://img.shields.io/badge/openhim--core-1.5%2B-brightgreen.svg)](http://openhim.readthedocs.org/en/latest/user-guide/versioning.html)
+[![Build Status](https://travis-ci.org/jembi/openhim-console.svg?branch=master)](https://travis-ci.org/jembi/openhim-console) [![OpenHIM Core](https://img.shields.io/badge/openhim--core-3.0.x-brightgreen.svg)](http://openhim.readthedocs.org/en/latest/user-guide/versioning.html)
```

Comment:
Maybe this link will be better if if took the user to the readthedocs version
e.g. http://openhim.readthedocs.io/en/v3.0.0/index.html